frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -102,11 +137,11 @@ export const InteractiveEmbeddingOptionCard = () => {
   return (
     <EmbeddingOption
       icon={
-        !isEE || enabled ? (
-          <InteractiveEmbeddingOn />
-        ) : (
-          <InteractiveEmbeddingOff />
-        )
+        <InteractiveEmbedding
+          className={cx(EmbeddingOptionStyle.icon, {
+            [EmbeddingOptionStyle.disabled]: isEE && !enabled,
```

Comment:
this icon is always active when not on EE. Here's the previous condition for displaying (opposite of disabling it)

https://github.com/metabase/metabase/blob/0af1d7494bce055b3949697a398eb052ed03257b/frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx#L105

This condition is the negation of the previous condition since it's for the disabled state.